asapdiscovery-data/asapdiscovery/data/schema_v2/target.py

```diff
@@ -138,6 +146,11 @@ class PreppedTarget(DataModelAbstractBase):
         description="Enum describing the data storage method",
         allow_mutation=False,
     )
+    target_hash: str = Field(
```

Comment:
Could this also be done for `Target` not just `PreppedTarget` just so the interface is consistent?